media/wav.ksy

```diff
@@ -556,10 +628,21 @@ enums:
     0x45564157: wave
     0x5453494c: list
     0x4f464e49: info
+    0x74636166: fact
     0x20746d66: fmt
-    0x74786562: bext
     0x20657563: cue
     0x61746164: data
     0x64696d75: umid
     0x666e696d: minf
     0x6e676572: regn
+    0x20336469: id3
+    0x4b414550: peak
+    0x584d505f: pmx
+    0x4c4d5869: ixml
```

Comment:
This also looks like a "BWF chunk" to me - from http://www.gallery.co.uk/ixml/:

> iXML is primarily designed to be used as a [RIFF](http://www.gallery.co.uk/ixml/glossary.html) (embedded tagged data) chunk inside a **Broadcast Wave file** (although it can be optionally included in other file types), to **supercede** the metadata currently written in the **standard Broadcast Wave 'bext' chunk** description field in a non-standardised way by several manufacturers.

Reply:
Yes, BWF (and related) are based on .wav. So the question is: should this be in the .wav format, or in a separate format that will be largely identical? If there would be some sort of inheritance mechanism in KS it would be a lot simpler.

Reply:
> So the question is: should this be in the .wav format, or in a separate format that will be largely identical? If there would be some sort of inheritance mechanism in KS it would be a lot simpler.

I think it should stay in the .wav format, that's completely OK. This is almost the same thing as PNG/APNG - https://github.com/kaitai-io/kaitai_struct_formats/blob/5f3ce8dd442557996d6a7370526ea1fe00848c90/image/png.ksy#L5-L6. APNG still has the exact same structure as PNG, as well as BWF has the same structure as WAV, so keeping them in the same spec makes sense to me.

Reply:
Instead, I had in mind that if `ixml` is rather a BWF chunk, it should be below this comment

https://github.com/kaitai-io/kaitai_struct_formats/blob/48e78529e6034c2c2e497aa027d92b747613197e/media/wav.ksy#L642-L644

which attempts to separate the BWF chunks from the "pure WAV" ones, as far as I understand.

Reply:
yes, makes sense